plugins/tpm-eventlog/fu-tpm-eventlog-device.c

```diff
@@ -48,7 +48,11 @@ fu_tpm_eventlog_device_report_metadata(FuTpmEventlogDevice *self)
 	for (guint i = 0; i < self->items->len; i++) {
 		FuTpmEventlogItem *item = g_ptr_array_index(self->items, i);
 		g_autofree gchar *blobstr = fu_tpm_eventlog_blobstr(item->blob);
-		g_autofree gchar *checksum = fu_tpm_eventlog_strhex(item->checksum_sha1);
+		g_autofree gchar *checksum = NULL;
+		if (item->checksum_sha1)
+			checksum = fu_tpm_eventlog_strhex(item->checksum_sha1);
+		else if (item->checksum_sha256)
+			checksum = fu_tpm_eventlog_strhex(item->checksum_sha256);
```

Comment:
ohh good catch. Can you add a:

    else
        continue;

and then we can cover ourselves for the future.